ci/tasks/stop-rds-instance.yml

```diff
@@ -7,7 +7,7 @@ image_resource:
 params:
   RDS_INSTANCE_NAME:
   AWS_DEFAULT_REGION: eu-west-1
-  MAX_ATTEMPTS: 100
+  MAX_ATTEMPTS: 140 # 40 minutes
```

Comment:
Over the weekend there was a failure where it hadn't stopped in the 25 minutes, so give it a bit longer by default